src/components/RadialMenu.svelte

```diff
@@ -109,17 +109,26 @@
 		return `--rotate: ${rotate}deg`;
 	}
 
+	function setBodyUserSelect(val: string) {
```

Comment:
I'd personally put this in a helper file. Maybe `src/helpers/style.ts`?

Reply:
Fixed in HEAD